README.md

```diff
@@ -14,13 +14,16 @@ For more information please refer to the main [Apiary](https://github.com/Expedi
 | ct\_common\_config\_yaml | Common Circus Train configuration to be passed to internal Circus Train instance. It can be used, for example to configure Graphite for Circus Train. Refer to [Circus Train README](https://github.com/HotelsDotCom/circus-train/blob/master/README.md) for an exhaustive list of options supported by Circus Train. | string | n/a | yes |
 | docker\_image | Full path of Shunting Yard Docker image. | string | n/a | yes |
 | docker\_version | Shunting Yard Docker image version. | string | n/a | yes |
+| docker\_registry\_auth\_secret\_name | Docker Registry authentication SecretManager secret name. | string | `` | no |
 | instance\_name | Shunting Yard instance name to identify resources in multi-instance deployments. | string | `""` | no |
 | memory | The amount of memory (in MiB) allocated to the Shunting Yard container. Valid values: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html | string | `"4096"` | no |
 | metastore\_events\_sns\_topic | SNS Topic for Hive Metastore events. | string | n/a | yes |
-| selected\_tables | Tables selected for Shunting Yard Replication. Supported Format: `database_1.table_1, database_2.table_2` | string | n/a | yes |
+| shuntingyard_sqs_queue_wait_timeout | Shunting Yard SQS queue wait timeout (in seconds) | string | 15 | no |
```

Comment:
Sorry to be so anal about this but why do these not have `\` before the `_`? We should have everything consistent within this file (and project) one way or the other. I prefer not having the `\` if it's not actually needed.

Reply:
Ok. I have added the `\` to the new fields.

Reply:
This section of the readme is generated using `terraform-docs` tool which adds these `\` to the fields.